requirements-test.txt

```diff
@@ -1,3 +1,3 @@
 coveralls
 django-extensions>=2.0.7
-openwisp-utils[qa]~=1.0.1
+openwisp-utils[qa]~=1.0.2
```

Comment:
Why? This will be incompatible with https://github.com/openwisp/openwisp-monitoring/blob/master/requirements.txt#L7

1.0.2 does not contain all the changes in the master branch of openwisp-utils, it contains only the bugfixes, but in the new version of the OpenWISP modules we'll need the new featurs.